src/ape/types/signatures.py

```diff
@@ -1,11 +1,11 @@
 from typing import Iterator, Union
 
-from dataclassy import dataclass
 from eth_account.messages import SignableMessage
 from eth_utils import to_bytes
+from pydantic.dataclasses import dataclass
 
 
-@dataclass(frozen=True, slots=True, kwargs=True)
```

Comment:
I am curious about the original intention here for the `kwargs` and `slots` specs.
From my understand, all Pydantic models have `__slots__` so we are good to remove that (However there may be a bug when using Cython?: https://github.com/samuelcolvin/pydantic/issues/3791)

But, `kwargs`, why were using that to begin with? @fubuloubu ?

Reply:
I think `kwargs` just allows you to specify names via `v=....`